openapi/googleapis.cloudbuild/Package.md

```diff
@@ -0,0 +1,19 @@
+Connects to Google Cloud Build from Ballerina
```

Comment:
Should have a link to API `[Google Cloud Build](https://cloud.google.com/build/docs/api/reference/rest)`

Reply:
Fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/eaf8ead8f3f4d589ca6b3dbebd4278a3119fe129